src/js/apps/patients/sidebar/action-sidebar_app.js

```diff
@@ -224,5 +232,26 @@ export default App.extend(extend({
   onRemoveAttachment(model) {
     model.destroy();
   },
+  addSubscriptions() {
+    const commentSubscriptions = [];
+
+    this.activityCollection.each(model => {
+      if (model.get('message')) {
```

Comment:
The activity collection items don't contain a `type` attribute for comments. The only way to tell if it's a comment is if it contains the `message` attribute.

Reply:
`model.type` I believe